ui/admin/tests/acceptance/credential-store/credentials/read-test.js

```diff
@@ -39,38 +43,67 @@ module('Acceptance | credential-stores | credentials | read', function (hooks) {
       scope: instances.scopes.project,
       type: 'static',
     });
-    instances.credential = this.server.create('credential', {
+    instances.usernamePasswordCredential = this.server.create('credential', {
       scope: instances.scopes.project,
       credentialStore: instances.staticCredentialStore,
       type: 'username_password',
     });
+    instances.usernameKeyPairCredential = this.server.create('credential', {
+      scope: instances.scopes.project,
+      credentialStore: instances.staticCredentialStore,
+      type: 'ssh_private_key',
+    });
     // Generate route URLs for resources
     urls.projectScope = `/scopes/${instances.scopes.project.id}`;
     urls.credentialStores = `${urls.projectScope}/credential-stores`;
     urls.staticCredentialStore = `${urls.credentialStores}/${instances.staticCredentialStore.id}`;
     urls.credentials = `${urls.staticCredentialStore}/credentials`;
-    urls.credential = `${urls.credentials}/${instances.credential.id}`;
+    urls.usernamePasswordCredential = `${urls.credentials}/${instances.usernamePasswordCredential.id}`;
+    urls.usernameKeyPairCredential = `${urls.credentials}/${instances.usernameKeyPairCredential.id}`;
     urls.unknownCredential = `${urls.credentials}/foo`;
     authenticateSession({});
   });
 
-  test('visiting /credentials', async function (assert) {
+  test('visiting username & password credential', async function (assert) {
+    assert.expect(2);
+    await visit(urls.staticCredentialStore);
+    await click(`[href="${urls.credentials}"]`);
+    await a11yAudit();
+    assert.strictEqual(currentURL(), urls.credentials);
+    await click(`[href="${urls.usernamePasswordCredential}"]`);
+    await a11yAudit();
+    assert.strictEqual(currentURL(), urls.usernamePasswordCredential);
+  });
+
+  test('visiting username & key pair credential', async function (assert) {
     assert.expect(2);
     await visit(urls.staticCredentialStore);
     await click(`[href="${urls.credentials}"]`);
     await a11yAudit();
     assert.strictEqual(currentURL(), urls.credentials);
-    await click(`[href="${urls.credential}"]`);
+    await click(`[href="${urls.usernameKeyPairCredential}"]`);
     await a11yAudit();
-    assert.strictEqual(currentURL(), urls.credential);
+    assert.strictEqual(currentURL(), urls.usernameKeyPairCredential);
+  });
+
+  test('cannot navigate to a username & password credential form without proper authorization', async function (assert) {
+    assert.expect(1);
+    instances.usernamePasswordCredential.authorized_actions =
+      instances.usernamePasswordCredential.authorized_actions.filter(
+        (item) => item != 'read'
+      );
+    await visit(urls.credentials);
+    assert.notOk(find(`[href="${urls.usernamePasswordCredential}"]`));
```

Comment:
Not sure if we're moving towards these but as Cameron suggested in the other PR, I personally do like 
```suggestion
    assert.dom(`[href="${urls.usernamePasswordCredential}"]`).doesNotExist();
```